plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,124 @@
+package stomp
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/go-stomp/stomp"
+
+	"github.com/influxdata/telegraf"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+type STOMP struct {
+	Host      string          `toml:"host"`
+	Username  string          `toml:"username"`
+	Password  string          `toml:"password"`
+	QueueName string          `toml:"queueName"`
+	Log       telegraf.Logger `toml:"-"`
+
+	HeartBeatSend time.Duration `toml:"heartBeatSend"`
+	HeartBeatRec  time.Duration `toml:"heartBeatRec"`
```

Comment:
These should be `config.Duration` to allow time suffices like "s", "ms" or "m" for seconds, milliseconds, minutes etc. Furthermore, we usually use "snake-case" in TOML, so you maybe want to change to
```suggestion
	HeartBeatSend config.Duration `toml:"heartbeat_timeout_send`
	HeartBeatRec  config.Duration `toml:"heartbeat_timeout_receive"`
```